lifetimes/utils.py

```diff
@@ -251,3 +251,34 @@ def check_all_frequency_values_are_integer_values(frequency):
     check_recency_is_less_than_T(recency, T)
     check_frequency_of_zero_implies_recency_of_zero(frequency, recency)
     check_all_frequency_values_are_integer_values(frequency)
+
+
+def customer_lifetime_value(transaction_prediction_model, frequency, recency, T, monetary_value, time=12, discount_rate=1):
+    """
+    This method computes the average lifetime value for a group of one or more customers.
+        transaction_prediction_model: the model to predict future transactions, literature uses
+            pareto/ndb but we can also use a different model like bg
+        frequency: the frequency vector of customers' purchases (denoted x in literature).
+        recency: the recency vector of customers' purchases (denoted t_x in literature).
+        T: the vector of customers' age (time since first purchase)
+        monetary_value: the monetary value vector of customer's purchases (denoted m in literature).
+        time: the lifetime expected for the user in months. Default: 12
+        discount_rate: the monthly adjusted discount rate. Default: 1
+
+    Returns:
+        Series object with customer ids as index and the estimated customer lifetime values as values
+    """
+    try:
+        df = pd.DataFrame(index=frequency.index)
+    except AttributeError: # the case when we're passing floats or ints instead of pd.Series
+        df = pd.DataFrame(index=[0])
```

Comment:
Outside of convenience during testing/prototyping, not really. I'll remove the try/except, because you're right that the docs indicate that the inputs should be vectors anyway.